packages/nx/src/utils/find-matching-projects.spec.ts

```diff
@@ -158,6 +159,16 @@ describe('findMatchingProjects', () => {
       'nested',
     ]);
   });
+
+  it('should do something when pattern is not matched', () => {
```

Comment:
The test case title doesn't seem correct. The "do something" bit is vague and would seem like the function should perform a side effect. Probably better to phrase it as what's expected to handle/return.

Also, this test case is inside a suite for the `findMatchingProjects` but the test case is testing a different function. Might be better to have it on a different test suite or on its own, standalone test case.

Reply:
oh yes, that is a leftover piece, i can remove that test entirely. I wanted to check what would be returned for certain strings 🤦‍♂️